### DISCUSSION TITLE
Serverless Plugin

### DISCUSSION THREAD

**Discussion by Gabe1203** [serverless/serverless#11095] [Q&A] [answered]
We recently updated to the most recent version of serverless to utilize serverless compose. Now when i try to run `serverless plugin install ...` I get: `Unknown service "plugin"`. Is this a bug?

**Comment by pgrzesik** [comment, marked as answer]
Hey @Gabe1203 👋 In what context are you running `plugin install` command? It cannot be run in directory with `serverless-compose.yml` as it then detects it as Compose project, not a Framework project and plugins are not supported on Compose level (they are supported on specific Framework projects as previously though)

**Reply by Gabe1203** [reply]
Got it - it was in a dir with serverless compose. Thanks for the help! Is there a recommended way on installing plugins when serverless compose is there? I currently am just npm installing the plugins but curious if there is a way through serverless.

**Reply by pgrzesik** [reply]
There is - you could either run `sls plugin install...` inside specific project or run `sls <name-of-service>:plugin:install ...` command in the context of Compose project
